setup.cfg

```
@@ -1,2 +1,10 @@
 [metadata]
+name = scikit-image
+description = Image Processing in Python
+author = Stefan van der Walt
+author_email = stefan@sun.ac.za
+url = https://scikit-image.org
+git_url = https://github.com/scikit-image/scikit-image
+download_url = https://scikit-image.org/docs/stable/install.html
 license_file = LICENSE.txt
```

```suggestion
license_file = LICENSE.txt
license = Modified BSD
```
I would keep the old text as well. Including the line `license_file` was simply the only way I found to include the license file in the source distribution.

Didn't you want to include the README file in the wheel distribution?  You had added an other line to this in your other PR